src/mcts/search.cc

```diff
@@ -123,6 +123,26 @@ class MEvaluator {
     return m;
   }
 
+  float GetM(Node* child, float q) const {
+    if (!enabled_ || !parent_within_threshold_) return 0.0f;
+    const float child_m = child->GetM();
+    float m = std::clamp(m_slope_ * (child_m - parent_m_), -m_cap_, m_cap_);
+    // Microsoft compiler does not have a builtin for copysign and emits a
+    // library call which is too expensive for a hot path like this.
+#if defined(_MSC_VER)
```

Comment:
Would be nice to move that into `utils/misc.h` :)

Reply:
utils/fastmath.h - done.